code/game/machinery/computer/law.dm

```diff
@@ -6,10 +6,20 @@
 	icon_state = "command"
 	circuit = "/obj/item/weapon/circuitboard/aiupload"
 	var/mob/living/silicon/ai/current = null
+	var/mob/living/silicon/ai/occupant = null
 	var/opened = 0
 
 	light_color = "#555555"
 
+/obj/machinery/computer/aiupload/attackby(I as obj, user as mob)
+	if(istype(I, /obj/item/device/aicard))
+		if(stat & (NOPOWER|BROKEN))
+			to_chat(user, "This terminal isn't functioning right now, get it working!")
+			return
+		I:transfer_ai("AIUPLOAD","AICARD",src,user)
```

Comment:
no : please
you gotta cast this as an AI card, I know it adds a line but it's our coding standards